commands/id.go

```diff
@@ -22,15 +31,28 @@ var idCmd = &cmds.Command{
 		cmdkit.StringOption("format", "f", "Specify an output format"),
 	},
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		details, err := GetAPI(env).ID().Details()
-		if err != nil {
-			return err
+		addrs := GetPorcelainAPI(env).NetworkGetPeerAddresses()
+		hostID := GetPorcelainAPI(env).NetworkGetPeerID()
+
+		details := IDDetails{
+			Addresses: make([]ma.Multiaddr, len(addrs)),
+			ID:        hostID,
+			// TODO: fill out the other details
```

Comment:
fwiw i think we should not have speculative fields. it is so misleading. i would prefer we didn't have an agentversion or protocol version or pubkey if they are not used.

Reply:
I know this is not your bug but could you file an issue describing the fields you are taking out so that someone can go through and add them in the future?

Reply:
Are we ever going to add other fields here? Unless we have specific ones in mind, I'm not sure we should write an issue for it.